services/web-app/components/mdx/components.js

```diff
@@ -73,26 +62,26 @@ const components = {
   table: PageTable,
   pre: Code,
   /* Core Carbon components */
-  Button,
-  StructuredListWrapper,
-  StructuredListHead,
-  StructuredListRow,
-  StructuredListCell,
-  StructuredListBody,
-  UnorderedList,
-  ListItem,
-  Tag,
+  // Button,
+  // StructuredListWrapper,
+  // StructuredListHead,
+  // StructuredListRow,
+  // StructuredListCell,
+  // StructuredListBody,
+  // UnorderedList,
+  // ListItem,
+  // Tag,
```

Comment:
can we delete instead?
```suggestion
```

Reply:
Yeah, definitely. I'll make that change.